doc/changes_from_dnf4.7.rst

```diff
@@ -387,6 +387,9 @@ To address this, the functionality has been split into two configuration options
 
 Additionally, corresponding command-line options ``--skip-broken`` and ``--skip-unavailable`` have been introduced for commands where applicable.
 
+Deprecation of the ``metadata_timer_sync`` option
+-------------------------------------------------
+The ``metadata_timer_sync`` configuration option is now deprecated in favor of systemd timer settings.
```

Comment:
I would rather phrase is as "is now obsoleted by dnf5-makecache.timer systemd timer settings". The reason is that the option does nothing. "Deprecated" would mean that users still can use it and that it has an effect.

Reply:
Make sense, thanks!